specification.yaml

```diff
@@ -32,7 +32,7 @@ paths:
                 type: object
                 properties:
                   user_id:
-                    $ref: '#/components/schemas/UserID'
+                    $ref: "#/components/schemas/UserID"
```

Comment:
i'm pretty sure this is generated by some api thing, so we could ignore it the next time it's regenerated.